collectors/osv/collectors.py

```diff
@@ -100,8 +100,11 @@ def collect(self, osv_id: Union[str, None] = None) -> str:
             # Surface an exception when collecting an individual OSV vulnerability
             osv_vuln = self.fetch_osv_vuln_by_id(osv_id)
             osv_id, cve_ids, content = self.extract_content(osv_vuln)
-            self.save_snippet(osv_id, cve_ids, content)
-            return f"OSV collection for {osv_id} was successful."
+            created, updated = self.save_snippet(osv_id, cve_ids, content)
+            if (created, updated) == (0, 0):
+                return f"OSV collection for {osv_id} was not successful due to historical data."
```

Comment:
nitpick: I would probably rephrase the sentence so it is clear that the fetched data is the historical data and not that the reason is in some unrelated historical data :smile:

Reply:
I was thinking about putting it here or not. It is not currently in NVD; it contains only the following:
```
        if cve is not None:
            return f"NVD CVSS collection for {cve} completed"
```
So should I add a similar part to NVD too?